app/API/V1/Controllers/UserController.php

```diff
@@ -34,6 +49,85 @@ public function getTTConfig(): array
         ];
     }
 
+    /**
+     * @param SimpleEventContract $event
+     * @throws \Doctrine\DBAL\ConnectionException
+     */
+    public function onPostStore(SimpleEventContract $event):void
```

Comment:
The stuff below here will work for starters but we will swap out with Raven

Reply:
Copy as we discussed.